api/src/functionalTest/java/uk/gov/hmcts/payment/functional/AccountFunctionalTest.java

```diff
@@ -46,7 +46,7 @@ public void setUp() throws Exception {
         }
     }
 
-    @Test
+    @Test @Ignore
```

Comment:
Why are we ignoring this test method?

Reply:
@aravindan89 , this class was ignored completely(pls check the master). I have just got ignore tag into each test since serenity report is unable to report individual tests as ignored if class is ignored completely. Please let me know, if you need any more info